posthog/queries/test/test_trends.py

```diff
@@ -166,6 +168,210 @@ def test_trends_compare(self):
             self.assertEqual(no_compare_response[0]["labels"][5], "Thu. 2 January")
             self.assertEqual(no_compare_response[0]["data"][5], 1.0)
 
+        def _test_interval_filtering(self, dates: List[str], result, **filter_params):
+            person1 = person_factory(team_id=self.team.pk, distinct_ids=["person_1"], properties={"name": "John"})
+            for time in dates:
+                with freeze_time(time):
+                    event_factory(
+                        event="event_name", team=self.team, distinct_id="person_1", properties={"$browser": "Safari"},
+                    )
+
+            response = trends().run(Filter(data={**filter_params, "events": [{"id": "event_name"}]}), self.team,)
+            self.assertEqual(response, result)
+
+        def test_minute_interval(self):
+            self._test_interval_filtering(
+                dates=["2020-11-01 10:20:00", "2020-11-01 10:22:00", "2020-11-01 10:25:00"],
+                interval="minute",
+                date_from="2020-11-01 10:20:00",
+                date_to="2020-11-01 10:30:00",
+                result=[
```

Comment:
testing the entire object should be preferred because it will keep the result between clickhouse and postgres very consistent